griptapecli/commands/skatepark.py

```diff
@@ -94,7 +94,7 @@ def register(
     environment: list[tuple[str, str]],
 ) -> None:
     url = f"http://{host}:{port}/api/structures"
-    directory = directory.rstrip("/")
+    directory = os.path.abspath(directory.rstrip("/"))
```

Comment:
does this fix the "gotta run the server from the same dir as the repo" or is this independent

Reply:
Yes